[2023-03-25 13:33:26,281] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: rightmove_extraction_dag.print_projects scheduled__2023-02-09T00:00:00+00:00 [queued]>
[2023-03-25 13:33:26,293] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: rightmove_extraction_dag.print_projects scheduled__2023-02-09T00:00:00+00:00 [queued]>
[2023-03-25 13:33:26,306] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-03-25 13:33:26,308] {taskinstance.py:1357} INFO - Starting attempt 6 of 6
[2023-03-25 13:33:26,310] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-03-25 13:33:26,326] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): print_projects> on 2023-02-09 00:00:00+00:00
[2023-03-25 13:33:26,342] {standard_task_runner.py:52} INFO - Started process 958 to run task
[2023-03-25 13:33:26,347] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'rightmove_extraction_dag', 'print_projects', 'scheduled__2023-02-09T00:00:00+00:00', '--job-id', '75', '--raw', '--subdir', 'DAGS_FOLDER/rightmove_extraction_dag.py', '--cfg-path', '/tmp/tmpcbrl3cs2', '--error-file', '/tmp/tmpwjxf8r45']
[2023-03-25 13:33:26,353] {standard_task_runner.py:80} INFO - Job 75: Subtask print_projects
[2023-03-25 13:33:26,426] {task_command.py:369} INFO - Running <TaskInstance: rightmove_extraction_dag.print_projects scheduled__2023-02-09T00:00:00+00:00 [running]> on host 6614738016e4
[2023-03-25 13:33:26,508] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alex
AIRFLOW_CTX_DAG_ID=rightmove_extraction_dag
AIRFLOW_CTX_TASK_ID=print_projects
AIRFLOW_CTX_EXECUTION_DATE=2023-02-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=6
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-02-09T00:00:00+00:00
[2023-03-25 13:33:26,511] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-03-25 13:33:26,518] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl http://localhost:6800/listprojects.json']
[2023-03-25 13:33:26,560] {subprocess.py:85} INFO - Output:
[2023-03-25 13:33:26,619] {subprocess.py:92} INFO -   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[2023-03-25 13:33:26,621] {subprocess.py:92} INFO -                                  Dload  Upload   Total   Spent    Left  Speed
[2023-03-25 13:33:26,622] {subprocess.py:92} INFO -   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
[2023-03-25 13:33:26,624] {subprocess.py:92} INFO - curl: (7) Failed to connect to localhost port 6800: Connection refused
[2023-03-25 13:33:26,626] {subprocess.py:96} INFO - Command exited with return code 7
[2023-03-25 13:33:26,640] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/operators/bash.py", line 194, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 7.
[2023-03-25 13:33:26,643] {taskinstance.py:1395} INFO - Marking task as FAILED. dag_id=rightmove_extraction_dag, task_id=print_projects, execution_date=20230209T000000, start_date=20230325T133326, end_date=20230325T133326
[2023-03-25 13:33:26,663] {standard_task_runner.py:92} ERROR - Failed to execute job 75 for task print_projects (Bash command failed. The command returned a non-zero exit code 7.; 958)
[2023-03-25 13:33:26,686] {local_task_job.py:156} INFO - Task exited with return code 1
[2023-03-25 13:33:26,716] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
